src/Analyzers/Analyzers.csproj

```diff
@@ -2,30 +2,22 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeShared>false</IncludeShared>
```

Comment:
What does `<IncludeShared>` do?

Reply:
Suppresses the logic we have to including files from `Shared.csproj`